### ISSUE TITLE
is there additional dataseet to finetune Maskrcnn

### ISSUE THREAD

**Issue by hetolin** [mentian/object-deformnet#1]
Hi, It's really amazing work and the method your proposed boost the performance a lot.
But there are no 'camera' and 'can' categories in COCO dataset. Could I ask if you had added other additional datasets to finetune the maskrcnn model? Or could you share your method on how to finetune or train the maskrcnn model.

**Comment by mentian** [comment]
Thanks for your interest in our work.
We finetune the mask-rcnn model on CAMERA and REAL datasets. We have provided the segmentation results of the testsets. If necessary, we could release both the training code and the finetuned mask-rcnn model in the future.

**Comment by hetolin** [comment]
Thanks for your prompt reply. Thank you very much if you could release both the training code and the finetuned mask-rcnn model in the future.

**Comment by Patlon** [comment]
> Thanks for your interest in our work.
> We finetune the mask-rcnn model on CAMERA and REAL datasets. We have provided the segmentation results of the testsets. If necessary, we could release both the training code and the finetuned mask-rcnn model in the future.

hello，could you release both the training code and the finetuned mask-rcnn model?

**Comment by mentian** [comment]
Please download from [here](https://drive.google.com/file/d/1XhowmkNvEw4LGs0RtAmwWvOHkwRsYUAT/view?usp=sharing).

**Comment by Liuchongpei** [comment]
@mentian Hi, I want to run eval.py in the folder ‘mcrnn‘, but I have a problem with environment configuration. What are your **python, tensorflow and keras** versions?
